docs/analog/commands/UpdateRun.md

```diff
@@ -0,0 +1,51 @@
+# UpdateRun
+
+Command to run the update on the device.
+This command is part of update procedure.
+
+
+## Request
+
+### Format
+
+| Size | Type   | Field               |
+| ---- | ------ | ------------------- |
+| `1`  | `byte` | extra flag = `0x1f` |
+| `1`  | `byte` | command id = `0x2c` |
+| `1`  | `byte` | command size        |
+
+It's a command with a [three-bytes header](../message.md#command-with-a-three-bytes-header).
+
+### Parameters
+
+The command has no parameters.
+
+### Examples
+
+| Field        | Value | Hex    |
+| ------------ | ----- | ------ |
+| extra flag   | `31`  | `0x1f` |
+| command id   | `44`  | `0x2c` |
+| command size | `0`   | `0x00` |
+
+Message hex dump with LRC: `1f 2c 00 33`
+
+## Response
+
+### Format
+
+| Size         | Type   | Field             |
+| ------------ | ------ | ----------------- |
+| extra flag   | `31`   | `0x1f`            |
+| command id   | `44`   | `0x2c`            |
+| command size | `0`    | `0x00`            |
+
+It's a command with a [three-bytes header](../message.md#command-with-a-three-bytes-header).
+
+### Parameters
+
+The command has no parameters.
+
+### Examples
+
+Message hex dump with LRC: `1f 2c 00 33`
```

Comment:
would be nice to have a full example body

and in other commands

Reply:
Sorry, but that is the message hex dump with LRC. What do you mean by full example body?